skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -609,31 +536,32 @@ public function executeCorporate() {
 	 * @param: array $params
 	 */
 	public function executeHubs( $params ) {
-		global $wgSupressPageTitle;
+		$wg = $this->wg;
 
-		wfProfileIn( __METHOD__ );
-
-		$this->displaytitle = true;
 		// Leave this for now. To discuss do we want PageTitle
-		if ( $this->displaytitle ) {
-			$this->title = wfMessage( 'oasis-home' )->escaped();
-		}
+		$this->title = wfMessage( 'oasis-home' )->escaped();
 
-		// number of pages on this wiki
-		$this->tallyMsg = wfMessage( 'oasis-total-articles-mainpage', SiteStats::articles() )->parse();
-
-		if ( !empty( $wgSupressPageTitle ) ) {
+		if ( !empty( $wg->SupressPageTitle ) ) {
 			$this->title = '';
 		}
 
-		wfProfileOut( __METHOD__ );
-	}
-
-	static function onArticleSaveComplete( &$article, &$user, $text, $summary,
-		$minoredit, $watchthis, $sectionanchor, &$flags, $revision, &$status, $baseRevId ) {
-		global $wgMemc;
-		$wgMemc->delete( wfMemcKey( 'mOasisRecentRevisions2', $article->getTitle()->getArticleId() ) );
-		return true;
+		// temp styles loader for testing
+		$this->response->addAsset( 'wikiahubs_v3_scss' );
```

Comment:
Yep, QA have config file set on their application instance, so it should be safe to remove it now.